frontend/src/pages/explore/TabPhenotypeExplorer.vue

```diff
@@ -7,8 +7,19 @@
 <template>
   <AppSection>
     <!-- example -->
-    <AppButton text="Try an example" design="small" @click="doExample()" />
-
+    <div style="display: inline-block">
```

Comment:
You should just use the `<AppFlex>` component here for consistency, probably with `gap="tiny"` (or whichever option looks appropriate).

Reply:
Other than this, looks good.

Reply:
Thanks, that was why I asked you to review! I'll make the change.